src/app/meshes/views/MeshView.vue

```diff
@@ -1,6 +1,11 @@
 <template>
-  <RouteView>
+  <RouteView
+    v-slot="{route}"
+  >
     <AppView>
+      <template #title>
+        <h1>{{ route.params.mesh }}</h1>
+      </template>
```

Comment:
This change will also be removed before merge.

Reply:
I've decided to leave this here as I am currently working on these views and I may as well leave it in here instead of remove here and re-add back in again.